test-browser/commands/noWorkerErrorFor.js

```diff
@@ -0,0 +1,28 @@
+const EventEmitter = require('events')
+
+class NoWorkerErrorFor extends EventEmitter {
+  command (version, content) {
+    this.api.perform((done) => {
+      noWorkerErrorFor(this.api, version, content, () => {
+        done()
+        this.emit('complete')
+      })
+    })
+    return this
+  }
+}
+
+function noWorkerErrorFor (browser, version, done) {
+  browser
+    .clickLaunchIcon('solidity')
+    .setSolidityCompilerVersion(version)
+    .waitForElementPresent('*[data-id="compiledErrors"]')
```

Comment:
is everything recompiled after version is  changed?

Reply:
any ideas on how I can check that besides waiting ?

Reply:
if you run it locally you'll see it

Reply:
yes but is there a way to assert it?